dockers/docker-orchagent/ipinip.json.j2

```diff
@@ -56,15 +55,24 @@
             "ttl_mode":"pipe"
         },
         "OP": "SET"
-    }
+    }{% if ipv4_addresses %},
```

Comment:
I don't see {% endif %} for this if condition.

Reply:
I feel like if we don't have `ipv4_addresses`, and only have `ipv6_loopback_addresses`, the `,` is also required.

Reply:
> I don't see {% endif %} for this if condition.
Please check LINE#59, this check condition adds the `,` only if there are IPv4 decap terms.

Reply:
> I feel like if we don't have `ipv4_addresses`, and only have `ipv6_loopback_addresses`, the `,` is also required.

If we don't have `ipv4_addresses`, LINE#45 ~ LINE#69 will be skipped, and we will only have IPv6 tunnel && decap terms, there is no need to add the `,` as the tunnel now comes to the begining.

Reply:
Is the `else` at Line#59 to close the `if` at Line#45?